product-service/src/main/java/com/extremecoder/productservice/controller/ProductController.java

```diff
@@ -2,24 +2,23 @@
 
 import com.extremecoder.productservice.model.Product;
 import com.extremecoder.productservice.service.ProductService;
-import org.springframework.beans.factory.annotation.Autowired;
+import com.extremecoder.productservice.utils.UrlConstraint;
+import lombok.RequiredArgsConstructor;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 import java.util.List;
 
 @RestController
-@RequestMapping("/product")
```

Comment:
Bhai, according to last discussion we agreed upon that our root mapping will be `/product-service`. Please change accordingly.

Reply:
Actually this "/product-service" attached at every end point because it is use 'context-path: /product-service" application.properties already. And yes according to the last discussion we set our end point info this ticket description. and right now the application is behaving like this. please check once again. https://extremecoder.atlassian.net/jira/software/projects/PROD/boards/1?selectedIssue=PROD-12

Reply:
Oh i see. Thanks for your clarification bhai.